client/src/scss/_dialogBackground.scss

```diff
@@ -0,0 +1,24 @@
+// TODO: Colors and image are not final but you get the impression
+.dialog-bg-image {
```

Comment:
Der Hintergrund sollte dann den ganzen Viewport ausfüllen. Z.Zt. habe ich dann so einen dummen großen weißen, bzw. schwarzen Balken.

![grafik](https://user-images.githubusercontent.com/42967368/143891806-df804e3d-78d8-43ca-a987-3367913cb361.png)

Reply:
interessant, ist bei mir nicht. checke ich bei Gelegenheit. firefox?

Reply:
Firefox 94; Win 10

Reply:
ach jetzt weiß ich was du meinst. ja das ist noch ne generelle Baustelle in der app. hab schon mehrmals probiert das gut hinzubekommen, aber ist nicht so einfach. kommt auf jeden fall irgendwann:D